test/mock/createInstantSearch.ts

```diff
@@ -8,7 +8,7 @@ export const createInstantSearch = (
   args: Partial<InstantSearch> = {}
 ): InstantSearch => {
   const searchClient = createSearchClient();
-  const indexName = 'index_name';
+  const indexName = args.indexName === undefined ? 'indexName' : args.indexName;
```

Comment:
You can simplify this logic with the rest/spread.

https://github.com/algolia/instantsearch.js/blob/bf0497864d880e41007e80ac163d270fcc3f83a0/test/mock/createWidget.ts#L14